labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,8 +32,26 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
+  
+  // After realizing the sample inputs/outputs provide enough information
+  // to figure out the mapping, I checked Prof Ben's solution to make sure
+  // there isn't actually any sort of consistent set of operations
+  // we can generally apply to each input character to get the correct output character.
+  // My intention before seeing his solution was to hardcode the mapping by
+  // indicating the mapping for each individual character,
```

Comment:
That was my solution; constructing the map at runtime didn’t even occur to me until after I’d finished and I was looking at the sample solutions.